test/integration/consul-container/assets/Dockerfile-consul-dataplane

```diff
@@ -2,7 +2,22 @@
 # SPDX-License-Identifier: BUSL-1.1
 
 ARG CONSUL_DATAPLANE_IMAGE
+ARG CONSUL_IMAGE
+
+# Docker doesn't support expansion in COPY --copy, so we need to create an intermediate image.
+FROM ${CONSUL_IMAGE} as consul
 
 FROM ${CONSUL_DATAPLANE_IMAGE} as consuldataplane
-COPY --from=busybox:uclibc /bin/sh /bin/sh
-COPY --from=ghcr.io/tarampampam/curl:latest /bin/curl /bin/curl
\ No newline at end of file
+
+USER root
+
+RUN microdnf install -y iptables sudo nc \
```

Comment:
Can we add a note for something like:

On Mac M1s when TProxy is enabled, consul-dataplane that are spawned from this image (only used in consul-container integration tests) will terminate with the below error.  It is related to tproxy-startup.sh calling iptables SDK which then calls the underly iptables.  We are investigating how this works on M1s with consul-envoy images which do not have this problem.  For the time being tproxy tests on Mac M1s will fail locally but pass in CI.

Error setting up traffic redirection rules: failed to run command: /sbin/iptables -t nat -N CONSUL_PROXY_INBOUND, err: exit status 1, output: iptables: Failed to initialize nft: Protocol not supported